bci_tester/data.py

```diff
@@ -304,6 +304,21 @@ def create_BCI(
     default_entry_point=True,
 )
 
+PCP_CONTAINER = create_BCI(
+    build_tag=f"suse/pcp:5.2.2",
+    image_type="dockerfile",
+    available_versions=["15.3"],
+    extra_launch_args=[]
+    if DOCKER_SELECTED
+    else [
+        "--systemd",
+        "always",
+        "-p",
+        "44322:44322",
+    ],
+    default_entry_point=True,
+)
```

Comment:
I would suggest that you make this container a singleton as you must not launch two instances of it.
Also, I would suggest that you move the skipIf from test_pcp into this call via the `extra_marks` parameter.

```suggestion
PCP_CONTAINER = create_BCI(
    build_tag=f"suse/pcp:5.2.2",
    image_type="dockerfile",
    singleton=True,
    extra_marks=[pytest.mark.skipif(DOCKER_SELECTED, reason="only podman is supported")],
    extra_launch_args=[
        "--systemd",
        "always",
        "-p",
        "44322:44322",
    ],
    default_entry_point=True,
)
```